src/test/java/iudx/resource/server/authenticator/JwtAuthServiceImplTest.java

```diff
@@ -764,7 +862,42 @@ public void access4ProviderTokenEntitiesAPI(VertxTestContext testContext) {
     jwtData.setIat(1627408865);
     jwtData.setIid("rg:example.com/79e7bfa62fad6c765bac69154c2f24c94c95220a/resource-group");
     jwtData.setRole("provider");
-    jwtData.setCons(new JsonObject().put("access", new JsonArray().add("api")));
+    jwtData.setSub("userid");
+    JsonObject access = new JsonObject()
+            .put("api", new JsonObject().put("limit", 1000).put("unit", "number"))
+            .put("sub", new JsonObject().put("limit", 10000).put("unit", "MB"))
+            .put("file", new JsonObject().put("limit", 1000).put("unit", "number"))
+            .put("async", new JsonObject().put("limit", 10000).put("unit", 122));
+
+    JsonArray attrs = new JsonArray()
+            .add("trip_direction")
+            .add("trip_id")
+            .add("location")
+            .add("id")
+            .add("observationDateTime");
+
+    JsonObject cons = new JsonObject()
+            .put("access", access)
+            .put("attrs", attrs);
+
+    jwtData.setCons(cons);
+
+    JsonObject meteringCountRequest = new JsonObject();
+    meteringCountRequest.put("startTime", "endDateTime");
+    meteringCountRequest.put("endTime", "startDateTim");
+    meteringCountRequest.put("userid", "userid");
+    meteringCountRequest.put("resourceId", "resourceId");
+    meteringCountRequest.put("accessType", "api");
```

Comment:
Please take appropriate measures to avoid code duplication here as well

Reply:
ignored for now because we need to re write the test cases in future